products/jbrowse-react-linear-genome-view/stories/JBrowseLinearGenomeView.stories.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```

Comment:
I'd prefer to keep this on a line-by-line basis so linting picks up any stray logs used during development.

Reply:
I think it's useful to avoid having it in the example code shown on the webpage

Reply:
Ah, I see. I'm fine with leaving it this way, then.